MarketplaceService/MqMessages/UpdateUser.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace marketplaceservice.MqMessages
+{
+    public class UpdateUserMessage
+    {
+        public Guid Id { get; set; }
+        public string NewEmail { get; set; }
```

Comment:
Is dit de enige data die een user heeft die geupdate kan worden? Anders zou UpdateEmailMessage misschien een betere classnaam zijn.